persistence/sql/persister_session.go

```diff
@@ -106,34 +107,27 @@ func (p *Persister) ListSessions(ctx context.Context, active *bool, paginatorOpt
 			return sqlcon.HandleError(err)
 		}
 
-		if expandables.Has(session.ExpandSessionIdentity) {
-			for index := range s {
-				sess := &(s[index])
-
-				i, err := p.GetIdentity(ctx, sess.IdentityID, identity.ExpandDefault)
-				if err != nil {
-					return err
-				}
-
-				sess.Active = sess.IsActive()
```

Comment:
@kelkarajay here's one of those examples where it doesn't make sense to set IsActive in the persister. By moving it to MarshalJSON (which I have done in another PR), `Active` will always be returned correctly by the JSON API. In our Go code, everyone should instead is `IsActive()`.